src/ensembl/static/img/shared/icon-xlink.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Regarding the file name: although all files in the img/shared folder are icons, none has a name starting with the word `icon`.
Perhaps naming it `external-link.svg` will fit the naming pattern better?

Reply:
That's true and `external-link` is a better name. I've changed it.